virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -6,33 +6,25 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+{VirtProductName} is installed with a default internal pod network.
 
-To attach a virtual machine to an additional network:
+Depending on the type of networks you are using, it may be necessary to update your network node configuration.
 
-. Configure a Linux bridge.
-. Configure a bridge network attachment definition for a namespace in the web console or CLI.
-+
-[NOTE]
-====
-The network attachment definition must be in the same namespace as the pod or virtual machine.
-====
-. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
-** In the web console, create a NIC for a new or existing virtual machine.
-** In the CLI, include the network information in the virtual machine configuration.
+In order to use your virtual machines with external networks, you must create a Linux _network bridge_ to expose your network services outside the default internal pod network.
 
-[NOTE]
-====
-There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
-However, a network attachment definition provides a more efficient and more manageable configuration.
-====
+Once a network bridge is created, you can start virtual machines on the external network in the same manner as you start virtual machines attached to internal networks, by using PXE booting, for example.
 
-include::modules/virt-networking-glossary.adoc[leveloffset=+1]
+include::modules/virt-about-creating-network-bridges.adoc[leveloffset=+1]
 
-[id="virt-creating-linux-bridge"]
+[id="virt-configuring-linux-bridge"]
 == Configuring a Linux bridge
 
+.Prerequisites
+
+* xref:../../../virt/node_network/virt-updating-node-network-config.adoc#node_network/virt-updating-node-network-config[Update your node network configuration] if necessary.
```

Comment:
Is this prerequisite the same as the next line (creating bridge interface on nodes in cluster)? Or is there another network configuration that the user might have to update? If so, please state what that would be or give an example. "if necessary" might be too vague for the user.